scipy/stats/stats.py

```diff
@@ -209,7 +211,8 @@
            'tiecorrect', 'ranksums', 'kruskal', 'friedmanchisquare',
            'rankdata', 'rvs_ratio_uniforms',
            'combine_pvalues', 'wasserstein_distance', 'energy_distance',
-           'brunnermunzel', 'epps_singleton_2samp', 'cramervonmises']
+           'brunnermunzel', 'epps_singleton_2samp', 'cramervonmises',
+           'mann_whitney_u', 'mannwhitneyu2']
```

Comment:
Why not name these more consistently? Either both with or without underscores?

Reply:
Please see the top post. In the end, there will only be one, hopefully.

Reply:
And actually, the reason I'd prefer to replace `mannwhitneyu` is for the sake of consistency. Most hypothesis test in SciPy have no underscores, and I've been pushing for new hypothesis tests to stick to that convention.